src/legacy/server/logging/log_reporter.js

```diff
@@ -19,19 +19,30 @@
 
 import { Squeeze } from 'good-squeeze';
 import { createWriteStream as writeStr } from 'fs';
+import { PassThrough } from 'stream';
 
 import LogFormatJson from './log_format_json';
 import LogFormatString from './log_format_string';
 import { LogInterceptor } from './log_interceptor';
 
+// NOTE: legacy logger creates a new stream for each new access
```

Comment:
are we still doing writes to the previous logging streams?  should they be closed?

Reply:
@jbudz I think `getLoggerStream` is called per each plugin that's the problem and it is related with the way legacy logger was built. Do you have any other idea?

Reply:
nope just wanted to check, works for me